pkg/conversion/unversioned_test.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
@lavalamp, I'm not sure about the file name (unversioned_test.go). I'm happy to modify the file name if you have other idea.